### ISSUE TITLE
Add ability to 'forget' a character

### ISSUE THREAD

**Issue by EnmaPlays** [EthanCentaurai/TradeSkillInfo#7]
I ran into this after a rename, but there could be a number of reasons a player might want to remove a character from TSI's database. I apologize if I've just missed it, but I can't figure out a way to do so without simply deleting the whole thing and logging each character to recreate it. 

Some ideas...

(1) Dropdown in the config menu that is populated with every character in the database. Select one and have a pop-up confirmation to delete that characters' data.

(2) /tsi forget <character_name>

**Comment by SonGokuSSJ** [comment]
Yes, that would be a nice feature. I have the problem, that it shows my Characters on items, which do not have the profession anymore (I have 2 characters and at the beginning both were having Tailoring and Enchanting, but I switched one character from Tailoring and Enchanting to Jewelcrafting and Mining). I tried also uninstalling the addon with varable deletion (its an option in minion Addon Updater), but that did not help either. I also tried your newest Alpha version, without solving the problem. So please add some kind of character forget button.
